bucket/vtfedit.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.3.3",
+    "description": "A VMT/VTF viewer, editor, and creation tool.",
+    "homepage": "http://nemesis.thewavelength.net/index.php?c=169",

```

Comment:
![image](https://user-images.githubusercontent.com/13260377/92017780-8d302580-ed54-11ea-9b79-f6bc130e6581.png)

Reply:
Should there be no homepage since the official is down, or an archived version of it?

Reply:
If there is no active homepage it will not be added.